### PR TITLE
Hw23

### PR DESCRIPTION
**Pull request by andrjuxa201185**
@iliatcymbal PTAL

### CODE REVIEW

File: src/pages/homeAuth/HomeAuth.js [hunk 1]
```diff
@@ -1,20 +1,16 @@
 import { useState, useEffect } from 'react';
 import { Link } from 'react-router-dom';
 import './homeAuth.scss';
+import { server } from '../../services';
 
 export const HomeAuth = ({ user }) => {
-  const [name, setName] = useState('');
   const [numCategories, setNumCategories] = useState(0);
   const [numPublished, setNumPublished] = useState(0);
   const [numProducts, setNumProducts] = useState(0);
```
Comment (iliatcymbal) [inline src/pages/homeAuth/HomeAuth.js]: use one object, it is not necessary to keep every field from response in separate variable

File: src/pages/products/Products.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { EditableField } from '../../components/editableField';
+import { server } from '../../services';
+import './products.scss';
+
+const Product = (title, id) => (
```
Comment (iliatcymbal) [inline src/pages/products/Products.js]: One component per file

File: src/pages/products/Products.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    <div className="description">
+      <div className="setting">
+        <span>edit</span>
+        <span>X</span>
+      </div>
+      <Link to={`/products/:${id}`} className="img">To Product</Link>
+    </div>
+    <EditableField type="textarea" val={title} />
+  </li>
+);
```
Comment (iliatcymbal) [inline src/pages/products/Products.js]: http://i.imgur.com/lxv9poT.png. There should be an icon for empty products and image for products with `image` field

File: src/pages/products/Products.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      .then(setProducts);
+  }, []);
+
+  return (
+    <div>
+      <h3>Products</h3>
+      <ul className="products">
+        {
+          products.map(({ title, id }) => (
+            Product(title, id)
```
Comment (iliatcymbal) [inline src/pages/products/Products.js]: why don't you use jsx syntax? `<Product .../>`

File: src/pages/user/User.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+  const { email, firstName, lastName } = user;
+  return (
+    <div className="user">
+      <ul className="user">
+        <li key={email}>{email}</li>
+        <li key={firstName}>{firstName}</li>
+        <li key={lastName}>{lastName}</li>
+        <li key="password">password</li>
+        <li key="passwordAgain">password again</li>
```
Comment (iliatcymbal) [inline src/pages/user/User.js]: The Form component should be used here

**Comment by andrjuxa201185** [comment]
@iliatcymbal please check

File: src/pages/user/User.js [hunk 6]
```diff
@@ -1,19 +1,19 @@
...
-      <ul className="user">
-        <li key={email}>{email}</li>
-        <li key={firstName}>{firstName}</li>
-        <li key={lastName}>{lastName}</li>
-        <li key="password">password</li>
-        <li key="passwordAgain">password again</li>
-      </ul>
-      <button>SAVE</button>
-    </div>
+    <form className="form">
```
Comment (iliatcymbal) [inline src/pages/user/User.js]: You have `Form` component. Why do you create a new form?

**Pull request merged**